noxfiles/utils_nox.py

```diff
@@ -30,6 +33,23 @@
 )
 COMPOSE_DOWN_VOLUMES = COMPOSE_DOWN + ("--volumes",)
 
+# TODO: this doesn't actually work to teardown active containers. To see this:
+# 1) Run `nox -s "fides_env(test) -- keep_alive`
+# 2) Run `nox -s teardown`
```

Comment:
this is silently erroring because the command is incorrect, it is expecting the `.env` file and it isn't finding it

Working on a fix

Reply:
Ah, my mistake - I found a reasonable solve for this: https://github.com/ethyca/fides/pull/3017/commits/12e60e64ec213f533368fd2e1184316b2b94fe70